templates/taxi/car_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create"}} car</h1>

```

Comment:
The `yesno` filter is used to determine whether the form is for updating or creating a car. Ensure that the `object` context variable is correctly set in the view to reflect the current operation.